AltUseVendors/__init__.py

```diff
@@ -73,6 +73,29 @@ class ShopInfo:
 VIAL_HEAL_PERCENT: float = 0.25
 
 
+# Sound AkEvents for playing
+AKE_BUY: str = "Ake_UI.UI_Vending.Ak_Play_UI_Vending_Buy"
+AKE_SELL: str = "Ake_UI.UI_Vending.Ak_Play_UI_Vending_Sell"
+AKE_DICT = {
+    Game.BL2: {
+        "InteractiveObj_VendingMachine_GrenadesAndAmmo": "Ake_VOCT_Contextual.Ak_Play_VOCT_Marcus_Vending_Munition_Purchase",
+        "InteractiveObj_VendingMachine_HealthItems": "Ake_VOCT_Contextual.Ak_Play_VOCT_Zed_Store_Welcome",
+        #"VendingMachine_Weapons_Definition": "Ake_VOCT_Contextual.Ak_Play_VOCT_Marcus_Vending_Munition_Bye",
+        # DLC Vendors
+        "IO_Aster_VendingMachine_GrenadesAndAmmo": "Ake_Aster_VO.VOCT.Ak_Play_VOCT_Aster_Marcus_Store_Purchase",
+        "IO_Aster_VendingMachine_HealthItems": "Ake_Aster_VO.VOCT.Ak_Play_VOCT_Aster_Zed_Store_Purchase",
+        #"VendingMachine_Aster_Weapons_Definition": "Ake_Aster_VO.VOCT.Ak_Play_VOCT_Aster_Marcus_Store_Bye",
+        #"VendingMachine_TorgueToken": "Ake_Iris_VO.Ak_Play_Iris_TorgueVendingMachine_Purchase",
+    },
+    Game.TPS: {
+        "InteractiveObj_VendingMachine_GrenadesAndAmmo": "Ake_VOCT_Contextual.Ak_Play_VOCT_Marcus_Vending_Munition_Purchase",
+        "InteractiveObj_VendingMachine_HealthItems": "Ake_Cork_VOCT_Contextuals.Cork_VOCT_NurseNina.Ak_Play_VOCT_Cork_NurseNina_Store_Purchase",
+    },
+    Game.AoDK: {
+        # Don't know other game AkEvent names
```

Comment:
open blcmm has aodk dumps, at a guess it'll just be the aster ones

Reply:
Found them and also added in TPS DLC ones.